scRNA-seq-advanced/setup/intro-diff-expression-setup.Rmd

```diff
@@ -0,0 +1,624 @@
+                       colour_by = "celltype_broad",
+                       point_size= 0.5,  
+                       point_alpha = 0.4) +
+  facet_wrap(~ rms_sce$batch)
+```
+
+It looks like the pink cells, or the `Tumor_Myoblast` are the most dominant cell type across the libraries. 
+This might be an interesting group to start with for differential expression. 
+We can confirm this by looking at the total numbers of cells in each cell type. 
+
+```{r}
+table(rms_sce$celltype_broad)
+
+# can also print out a summary of the cell types across each library 
+colData(rms_sce) %>%
+  as.data.frame() %>%
+  dplyr::count(batch, celltype_broad) %>%
+  dplyr::group_split(batch)
+```
+
+`Tumor_Myoblast` is definitely the top represented cell type across the libraries. 
+So we want to look at myoblasts across our libraries and see if there are any changes in the myoblast population between samples of different types. 
+To do this, we should probably look at our sample metadata to see what types of samples we are looking at. 
+
+```{r}
+# Let's look at some of the clinical sample metadata that we have 
+table(rms_sce$diagnosis) # everything is RMS 
+table(rms_sce$subdiagnosis) # sample is split between ERMS and ARMS 
+
+# how many of each, 3 of each subdiagnosis, that's enough replicates so let's compare! 
+colData(rms_sce) %>%
+  as.data.frame() %>%
+  dplyr::count(batch, subdiagnosis)
+```
+
+It looks like we have three samples that are from ARMS patients and three samples that are from ERMS patients. 
+A good question we could ask is if there are differences in gene expression patterns in the myoblast population between ARMS and ERMS patients. 
+If we want to ask that question, then we need to have replicates, e.g. more than one sample for each type of patient, which we do! 
+
+Let's go ahead and try it! 
+
+## Differential Expression between conditions 
+
+### Pseudo-bulking 
+
+Before we can do DE analysis to compare the gene expression profiles of myoblasts in ARMS vs. ERMS patients, we will need to "pseudo-bulk" the gene counts. 
+Pseudo-bulking the counts results in a new counts matrix that has been produced from summing the counts together for all cells with the same combination of label (e.g. cell type) and sample. 
+This allows us to use single-cell resolution to define the labels and sum gene counts across groups of cells containing the same label, and avoids counting each cell as its own replicate. 
+
+Below are a few reasons why pseudo-bulking is helpful: 
+
+- Produces larger counts which are more amenable to standard normalization and differential expression methods used by bulk RNA-sequencing. 
+- Collapses gene expression counts by sample, reflecting that each sample is a replicate rather than each cell. 
+- Masks variance within a sample and only accounts for if a gene is uniformly high or low across samples. 
+
+Let's start with the simple comparison of looking at one cell type and comparing across ERMS and ARMS subtypes. 
```

Comment:
I think for the actual notebook, we want to be sure to always frame these DE tests from a specific question/hypothesis standpoint instead of more exploratory vibes (which are appropriate here)!